portfolio/src/main/webapp/index.html

```diff
@@ -7,14 +7,25 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-      <img src="/images/B3913B16-73F9-41BC-BACE-6312C1DBCBA7.JPG" />
+    <nav>
+      <ul>
+          <li><a href='index.html'><strong>Rix</strong></a></li>
+          <li><a href='pages/AboutMe.html'>About Me</a></li>
+          <li><a href='pages/Projects.html'>Projects</a></li>
+          <li><a href='pages/Connect.html'>Connect</a></li>
+          <li><a href='pages/Pictures.html'>Pictures</a></li>
+          <li><a href='pages/ContactMe.html'>Contact Me</a></li>
+          <li><a href='pages/PlayMe.html'>Interactive</a></li>
+      </ul>
+    </nav>
+    
+    <div style="margin-top:25px">
+      <h1>Welcome<br></h1>
+      <div style="margin-left:40px;margin-right:40px">
+      <p><br>So happy to see you here! Please feel free to explore and learn a little bit more about me and my work.<br><br></p>
+        <img id="aboutMe" src="images/rix_trees.jpeg" alt="Girl with nature background">
+            <p> <br><br>I'm Rix, and I'm an undergraduate student pursuing a Computer Science major and a Mathematics minor at Texas Christian University. I'm a rising Junior, and I'm a STEM Scholar there-- and I'm super involved on campus! I'm a proud Charter sister of Kappa Delta Chi, and love to (1) Learn about technology, and (2) empower women. I love it most when both of these overlap!
```

Comment:
There's a HTML character entity (`&mdash;`) for a typographic "em" dash instead of typing --. It will stop the browser from (for example) deciding to line-break between the `-` characters.

Reply:
Also: the benefit of more line breaking is that it makes it easier to associate a comment with a specific line.